tcms/kiwi_auth/views.py

```diff
@@ -22,15 +22,14 @@
 User = get_user_model()  # pylint: disable=invalid-name
 
 
-class LoginViewWithCustomTemplate(views.LoginView):
+class LoginViewWithCustomTemplate(views.LoginView):  # pylint: disable=missing-permission-required
     def get_template_names(self):
         return ['registration/custom_login.html', 'registration/login.html']
 
 
-class PasswordResetView(views.PasswordResetView):
+class PasswordResetView(views.PasswordResetView):  # pylint: disable=missing-permission-required
     form_class = forms.PasswordResetForm
 
-
```

Comment:
Removing this blank line caused flake8 to fail. Restore it and we should be good to go.